### PR TITLE
D-Link DCS 930L and 932L auth bypass.

### PR DESCRIPTION
**Pull request by dincaus**

### CODE REVIEW

File: routersploit/modules/exploits/dlink/dcs_930l_932l_auth_bypass.py [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+    D-Link DCS web cameras allow unauthenticated attackers to obtain the
+    configuration of the device remotely. A copy of the device configuration can be
+    obtained by accessing unprocteted URL.
+    """
+    __info__ = {
+        'name': 'D-Link DCS Cameras Authentication Bypass',
+        'description': 'D-Link DCS web cameras allow unauthenticated attackers to obtain the configuration of the device remotely.'
+                       'A copy of the device configuration can be obtained by accessing unprocteted URL.',
+        'authors': [
+            'Roberto Paleari',  # vulnerability discovery
```
Comment (lucyoa) [inline routersploit/modules/exploits/dlink/dcs_930l_932l_auth_bypass.py]: Feel free to add your name as a routersploit module author "# routersploit module" or ignore it if your are Roberto :)

File: routersploit/modules/exploits/dlink/dcs_930l_932l_auth_bypass.py [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+
+class Exploit(exploits.Exploit):
+    """
+    D-Link DCS web cameras allow unauthenticated attackers to obtain the
+    configuration of the device remotely. A copy of the device configuration can be
+    obtained by accessing unprocteted URL.
+    """
+    __info__ = {
+        'name': 'D-Link DCS Cameras Authentication Bypass',
+        'description': 'D-Link DCS web cameras allow unauthenticated attackers to obtain the configuration of the device remotely.'
```
Comment (lucyoa) [inline routersploit/modules/exploits/dlink/dcs_930l_932l_auth_bypass.py]: We need space after period. On "show info" these sentences get concatenated

**Comment by lucyoa** [comment]
LGTM

**Comment by ThePentesterDiaries** [comment]
Awesome!

**Pull request merged**